src/components/Main/Main.jsx

```diff
@@ -0,0 +1,13 @@
+import { Outlet } from "react-router-dom";
+
+
+
+function MainContent() {
+	return (
+		<div>
+			<Outlet />
+		</div>
+	);
+}
+
+export default MainContent;
```

Comment:
This file is called `Main` but the function is `MainContent` - also this is done in a slightly different way to Home and About - the end result is the same, it's just a little inconsistent